pkg/policy/execute.go

```diff
@@ -245,15 +245,24 @@ func (e *Executor) Execute(ctx context.Context, req *ExecuteRequest, policy *Pol
 	return &total, nil
 }
 
-func (e *Executor) createCheckResult(ctx context.Context, selector []string, policyExecution *state.PolicyExecution, q *Check, qr *QueryResult, err error) error {
+func normalizeCheckSelector(policyExecution *state.PolicyExecution, policyPath []string, policyName string) string {
```

Comment:
Bare functions (`normalizeCheckSelector`) should be towards the end of the file

Reply:
fixed